core/src/components/select/select.tsx

```diff
@@ -772,12 +772,26 @@ export class Select implements ComponentInterface {
       this.open(ev);
     } else {
       /**
-       * Prevent clicks to the start/end slots from opening the select.
-       * We ensure the target isn't this element in case the select is slotted
-       * in, for example, an item. This would prevent the select from ever
-       * being opened since the element itself has slot="start"/"end".
```

Comment:
Can we ensure this context is preserved somewhere, whether in this specific comment or elsewhere (such as near the `target === this.el` check above)?

Reply:
re-added in https://github.com/ionic-team/ionic-framework/pull/28839/commits/03bd74dd9909f89b1fa8e76e7fcc4743bf1ffa2a